<?xml version="1.0" encoding="UTF-8"?>2017-09-12 15:54:01,938 - log.py[line:69] - INFO:   run 登录测试1.
2017-09-12 15:54:16,950 - remote_connection.py[line:478] - DEBUG: POST http://127.0.0.1:4723/wd/hub/session/a36a6a14-8956-4112-8b2d-dd2d26e1724a/context {"sessionId": "a36a6a14-8956-4112-8b2d-dd2d26e1724a", "name": "WEBVIEW"}
2017-09-12 15:54:17,105 - remote_connection.py[line:565] - DEBUG: Finished Request
2017-09-12 15:54:17,105 - Test_Case.py[line:65] - ERROR: Message: No such context found.
